docs/workflow operations/Optimize Doc.md

```diff
@@ -0,0 +1,107 @@
+# Optimize:
+## Goal
+Used to find the minimal or maximum value to a given parameter value or time that a QoI (Quality of Interest) constraint does not exceed the provided threshold.
+
+We can ask questions to try to find out a good start time for an intervention:
+
+I want to turn on "masking" (beta set to a smaller value ~0.1) 
+but I want to know how late I can do it while making the number of infected never exceed a specified number.
+
+We can use questions to try to find out parameter values:
+I want to turn on "quarantine" in day 30. How strong/effective (high beta needs to be) will this quarantine need to be in order to keep the number of infected lower than a (specified threshold) number?
```

Comment:
Maybe rephrase here,  as well as the similar copies below. We don't have context into what `beta` refers to here, so you want to spell out things in full.

E.g. "I want to turn on quarantine on day 30 for 90 days, represented by the variable xyz, how strong/weak will xyz need to be in order to keep the number of infected below ...."

Reply:
Thank you that is a good point. Especially if we are going to feed this documentation through llm it is best to be explicit and provide the full scope in one place rather than assume we understand beta here is a parameter that has some impact on infected